client/globals.d.ts

```diff
@@ -117,6 +118,7 @@ declare global {
 			task_badge?: string;
 		};
 		isWooPayStoreCountryAvailable: boolean;
+		wooStoreId: string;
```

Comment:
We can remove this now, since we're not using it and relying on Woo to provide the ID in the tracking.